contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -489,10 +496,11 @@ attachPLtsql_fragment(ParseTree *node, PLtsql_stmt *fragment)
 {
 	if (fragment)
 	{
-		const char *tsqlDesc = pltsql_stmt_typename(fragment);
-
 		if (pltsql_enable_antlr_detailed_log)
+		{
+			const char *tsqlDesc = pltsql_stmt_typename(fragment);
```

Comment:
this change is just for scope cleanup, right?

Reply:
this line was previously above the test, so it was always executed, also when the detailed logging was not enabled. that made no sense.